config/initializers/fog_setting.rb

```diff
@@ -1,11 +1,19 @@
  CarrierWave.configure do |config|
-     config.fog_credentials = {
+    # if Rails.env.development?
+    #   config.storage = :file
+    #   config.enable_processing = false
+    # else
+    # config.storage = :fog
```

Comment:
Any reason this is commented? I would prefer if it fell back to `:file` when in development.